mailpoet/assets/js/src/newsletter_editor/initializer.jsx

```diff
@@ -138,6 +113,19 @@ const initializeEditor = (config) => {
         newsletterTypesWithActivation.includes(newsletter.type) &&
         newsletter.status === 'active'
       ) {
+        // Users can confirm deactivation before visiting the page
```

Comment:
Should we add the check [before pausing newsletters ](https://github.com/mailpoet/mailpoet/pull/4785/files#diff-4e317ae921510cd1cac6b3bf3f58e7b6cc9cc5af674f9e752e56c720173a73faL118)that are sending?

Reply:
Great point, @brezocordero! I do think it's worth it to cover that edge case, and I've implemented it in fb00cf094fdf788fb663791ecb0c6b7b4b6991e6. I also changed the success notice to a system (i.e. warning) so it won't auto-dismiss. Let me know if it looks ok or if you think of anything else that could improve this PR.